courses/free-courses-en.md

```diff
@@ -845,6 +845,7 @@
 * [Nuxt.js Fundamentals](https://vueschool.io/courses/nuxtjs-fundamentals)
 * [Vue JS 3 Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgeQIfSDtEGVvvSEPDkL_1f) - Vishwas Gopinath (Codevolution)
 * [Vue Router for Everyone](https://vueschool.io/courses/vue-router-for-everyone)
+* [Vue.js 3 state managment with pinia](https://youtu.be/Ok6vO98RV_Q)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links

Reply:
Hi, thanks for the comments, I've already done the changes